### PR TITLE
Added unit tests for query_repo_url and query_repository in mozci

### PR DESCRIPTION
**Pull request by adusca**
I started with the simplest functions. This is mostly testing the waters for future tests (especially in triggers).

### CODE REVIEW

**Comment by armenzg** [comment]
This looks good so far.

Could you please rebase? I want to see the coverage comment in here so we can see what the improvement looks like.

Do you want me to merge it once we rebase?

**Comment by vaibhavmagarwal** [comment]
@adusca , can you have docstring/comments for the tests that you create. It helps to look at the test and see what it does rather than to figure it out. Also, we can maybe use the docstrings further in sphinx for documentation.

**Comment by armenzg** [comment]
I wouldn't see us using documentation for tests, however, I agree that we need docstrings.

**Comment by armenzg** [comment]
Thanks for bringing it up!

File: test/test_mozci.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import pytest
+
+import mozci.mozci
+import mozci.sources
+
+MOCK_JSON = '''{
+            "real-repo": {
+            "repo": "https://hg.mozilla.org/integration/real-repo",
+            "graph_branches": ["Real-Repo"],
+            "repo_type": "hg"}}'''
```
Comment (vaibhavmagarwal) [inline test/test_mozci.py]: Can you please ident this:
'''{
     "real-repo": {
                          "repo": "https://hg.mozilla.org/integration/real-repo",
                          "graph_branches": ["Real-Repo"],
                           "repo_type": "hg"
            }
}'''

Comment (vaibhavmagarwal) [inline test/test_mozci.py]: Hmm, I indented in the comment, but looks like it removed the white space. Anyway, please ident this :)

**Pull request merged**